File: 1d_Array/Cpp/No5597.cpp
#include <iostream>

using namespace std;

int main() {
    cin.tie(NULL);
    ios::sync_with_stdio(false);

    bool allStudents[31] = { false };
    int submittedStudents;
    for (int i = 0; i < 28; ++i) {
        cin >> submittedStudents;
        allStudents[submittedStudents] = true;
    }

    int missingStudents[2];
    int idx = 0;
    for (int i = 1; i <= 30; ++i) {
        if (!allStudents[i]) {
            missingStudents[idx++] = i;
            if (idx == 2) {
                break;
            }
        }
    }

    cout << missingStudents[0] << "\n";
    cout << missingStudents[1] << "\n";

    return 0;
}

File: 1d_Array/python/No5597.py
# 1부터 30까지의 출석번호를 가진 학생 목록을 준비
all_students = set(range(1, 31))

# 제출한 학생들의 출석번호를 입력 받음
submitted_students = set()

for _ in range(28):
    submitted_students.add(int(input()))

# 제출하지 않은 학생들을 찾음
missing_students = list(all_students - submitted_students)

# 오름차순으로 정렬 후 출력
missing_students.sort()
print(missing_students[0])
print(missing_students[1])
